### ISSUE TITLE
SWIG documentation is missing

### ISSUE THREAD

**Issue by Silur** [islog/liblogicalaccess#188]
in 2016 there was a swig-wrapper repo, and it's clear from the commits that some refactoring happened around SWIG but there is no documentation how to invoke SWIG on the current source.

Please add a few lines into the wiki?

**Comment by Maxhy** [comment]
Correct. We need to finish to split the SWIG library into two libraries then it will be published on GitHub and documented. It's taking some time I know but we are moving forward on this direction...
